### PR TITLE
Add `prefix` option

### LINKED ISSUE #10: Add prefix option
Like https://github.com/fregante/daily-version#prefix

```diff
Version:
    steps:
    - uses: actions/checkout@v2
    - name: Create tag if necessary
      uses: fregante/daily-version-action@v1
      with:
        prefix: v
```

Required in https://github.com/npmhub/npmhub/pull/103

### PR DESCRIPTION
**Pull request by notlmn**
Closes #10

### Test

Run result: https://github.com/notlmn-dump/github-playground/runs/902270310?check_suite_focus=true#step:3:4
Created tag: https://github.com/notlmn-dump/github-playground/releases/tag/v20.7.23

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -23,7 +23,8 @@ async function init() {
 	}
 
 	// A new tag must be created
-	const version = dailyVersion();
+	const prefix = core.getInput('prefix') || '';
+	const version = dailyVersion(prefix);
```
Comment (fregante) [inline index.js:27]:
```suggestion
	const version = dailyVersion(core.getInput('prefix'));
```

`core.getInput` returns an empty string anyway: https://github.com/actions/toolkit/blob/1cc56db0ff126f4d65aeb83798852e02a2c180c3/packages/core/src/core.ts#L76

Comment (fregante) [inline index.js]:
```suggestion
	const version = dailyVersion(core.getInput('prefix'));
```

`core.getInput` returns an empty string anyway: https://github.com/actions/toolkit/blob/1cc56db0ff126f4d65aeb83798852e02a2c180c3/packages/core/src/core.ts#L76

**Comment by fregante** [comment]
There seems to be a problem:




Run: https://github.com/npmhub/npmhub/actions/runs/183499191

Config: https://github.com/npmhub/npmhub/actions/runs/183499191/workflow#L27

Could it be that it's a number and not a string? Is it setup correctly? It looks like it expects an array

**Comment by notlmn** [comment]
> Could it be that it's a number and not a string? Is it setup correctly? It looks like it expects an array

That looks like a conversion issue. The issue here might be (and I'm speculating here) is that GitHub's YAML parser only allows strings for value part of input fields. Which is what it is trying to say over there, that it accepts a range of input types, in this case only one being string types.

**Comment by fregante** [comment]
I think that the input is not configured correctly actually. That `['']` is actually supposed to be a list of allowed input keys, not values. Example run:

https://github.com/microsoft/vscode/actions/runs/183404207

File: action.yml [hunk 2]
```diff
@@ -4,6 +4,9 @@ icon: calendar
 branding:
   icon: 'calendar'
   color: 'purple'
+inputs:
+  prefix:
+    description: 'String to prepend to the tag name'
```
Comment (fregante) [inline action.yml:9]: Could it be that we also need to specify either `required: true` or a default value?

```suggestion
    description: 'String to prepend to the tag name'
    default: '' 
```

Example: https://github.com/microsoft/vscode-github-triage-actions/blob/master/release-pipeline/action.yml

**Comment by notlmn** [comment]
@fregante that workflow file of npmhub you shared still has reference to `v1` of this action.

https://github.com/npmhub/npmhub/actions/runs/183499191/workflow#L23.

**Comment by fregante** [comment]
Sigh. Why does GitHub suggest tagging/replacing releases this way?

The workflow is correct, but `v1` should be a branch, not a tag

**Pull request merged**